about/steering_committee.md

```diff
@@ -6,7 +6,7 @@ description: OceanHackWeek Steering Committee
 
 # Steering Committee
 
-The OceanHackWeek Steering Committee helps chart the continued evolution of OceanHackWeek and focuses on longer term issues like funding and infrastructure.
+The OceanHackWeek Steering Committee helps chart the continued evolution of OceanHackWeek and focuses on longer term issues like funding and infrastructure. The [Steering Committee Charter](https://oceanhackweek.github.io/about/charter.html) guides these efforts.
```

Comment:
I believe that you can do a relative link `./charter`, so that it will work the same locally, in preview environments, and on the deployed version of the site (especially if we change domains, this will save needing to go through and replace old references).

```suggestion
The OceanHackWeek Steering Committee helps chart the continued evolution of OceanHackWeek and focuses on longer term issues like funding and infrastructure. The [Steering Committee Charter](./charter) guides these efforts.
```

Reply:
oooo nice